spec/games_spec.rb

```diff
@@ -61,4 +61,29 @@
       expect(game_report).to eq(expected_final_report)
     end
   end
+
+  describe '#death_report' do
+    subject(:games){described_class.new('spec/fixtures/example.log')}
+
+    it 'returns an array' do
+      game_report = subject.death_report
+
+      expect(game_report).to be_an(Array)
+    end
+
+    it 'returns expected death report' do
+      expected_death_report = [
```

Comment:
use let for refining variables in tests.
`let(:expected_death_report) { <code here> }`